app/controllers/idv/phone_errors_controller.rb

```diff
@@ -10,8 +10,12 @@ class PhoneErrorsController < ApplicationController
 
     def warning
       @remaining_attempts = throttle.remaining_count
-      @phone = idv_session.previous_phone_step_params[:phone]
-      @country_code = idv_session.previous_phone_step_params[:international_code]
+
+      if idv_session.previous_phone_step_params
```

Comment:
```suggestion
      if idv_session.&previous_phone_step_params
```

Reply:
Like Sonia said in her review comment, this does seem to fix the 500 and we can patch it. That said, I do not think this is a valid controller action if you haven't submitted a phone and we should 302 in that case.